src/main/java/org/zalando/nakadi/service/subscription/zk/NewZkSubscriptionClient.java

```diff
@@ -216,4 +217,8 @@ public void transfer(final String sessionId, final Collection<EventTypePartition
         }
     }
 
+    @Override
+    public void close() throws IOException {
+        getCloseableCuratorFramework().close();
+    }
```

Comment:
Very strange. Obtain client in AbstractClient, release - in NewZkClient.

Reply:
this is a mistake, I moved to abstract class